source/PlayerInfo.cpp

```diff
@@ -875,6 +840,53 @@ Account &PlayerInfo::Accounts()
 
 
 
+// Handle the daily salaries and payments.
+void PlayerInfo::DoAccounting()
+{
+	// Check what salaries and tribute the player receives.
+	int64_t salariesIncome = accounts.SalariesIncomeTotal();
+	int64_t tributeIncome = GetTributeTotal();
+	FleetBalance b = MaintenanceAndReturns();
```

Comment:
for future coders a name other than "b" that actually says what the variable is would be nice

Reply:
ohhh

Reply:
can we still do it?

Reply:
I guess?